catalog/transformer/catalog-transformer-microsoft/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+    <bean id="msWordTransformer" class="ddf.catalog.transformer.input.MicrosoftTransformer">
+        <argument ref="blueprintBundleContext"/>
+        <argument ref="clavinFacade"/>
+    </bean>
+    <service ref="msWordTransformer" interface="ddf.catalog.transform.InputTransformer">
+        <service-properties>
+            <entry key="id" value="msword"/>
+            <!-- shortname only exists for backwards compatibility -->
+            <entry key="shortname" value="msword"/>
+            <entry key="description"
+                   value="Detects and extracts metadata microsoft word documents."/>
+            <entry key="mime-type">
+                <list>
+                    <value>application/msword</value>
+                    <value>application/vnd.ms-excel</value>
+                    <value>application/vnd.ms-powerpoint</value>
```

Comment:
Are Excel and Powerpoint actually MS Word documents? Or are they MS Office documents? Does your code only work with old-style (pre openoffice xml) documents?